test/README.md

```diff
@@ -78,8 +78,7 @@ make localintegration RUNTIME=cc-oci-runtime
 
 ## Writing integration tests
 
-[Helper functions]
-(https://github.com/kubernetes-incubator/crio/blob/master/test/helpers.bash)
+[Helper functions](https://github.com/kubernetes-incubator/crio/blob/master/test/helpers.bash)
```

Comment:
there is another typo actually: `crio -> cri-o`

Reply:
done